.gitignore

```diff
@@ -200,3 +200,6 @@ scripts/release/release_body.txt
 
 # Personal symlink to semgrep-proprietary used by 'make pro'
 /semgrep-proprietary
+
+# alcotest_ext test results
+/_test_results/status
```

Comment:
would be better to have it work like alcotest and have the results in _build/_tests/ and the expected snapshot in a different directory one can configure, like tests/snapshots/

Reply:
Note that this is configurable already. The issue here is that `_build/_tests/ ` is already used by alcotest. Also, `_build` is used by dune so I'm not sure which names are ok for us to use without risking conflicts. Perhaps `_build/_alcotest_ext` would be fine or `_build/alcotest_ext`. Note also that I don't like the name `alcotest_ext`. It will be renamed at some point. Still, it's a good idea to use a directory that matches the framework's name. I think ideally the structure of `_build` should be:
```
_build/
  dune/
  alcotest/
  alcotest_ext/
```

Reply:
I changed this. The new defaults are `_build/alcotest_ext` and `tests/snapshots`.